pages/Rating/rating.css

```diff
@@ -0,0 +1,39 @@
+/* Slider Code */
+.rating-container input {
+      display: none;
+}
+
+.rating-container label {
+      display: inline-block;
+      width: 1rem;
+      position: relative;
+      cursor: pointer;
+      color: rgb(255, 123, 0);
```

Comment:
suggestion for using a variable instead of hard coding the color

Reply:
Thanks Ruchi! I will update it